tapis-jobslib/src/main/java/edu/utexas/tacc/tapis/jobs/launchers/AbstractJobLauncher.java

```diff
@@ -74,7 +74,7 @@ protected String getLaunchCommand()
         // directory and runs the wrapper script.  The directory is expressed
         // as an absolute path on the system.
         String execDir = JobExecutionUtils.getExecDir(_jobCtx, _job);
-        return String.format("cd %s;./%s", conditionalQuote(execDir), JobExecutionUtils.JOB_WRAPPER_SCRIPT);
+        return String.format("cd %s;./%s", alwaysSingleQuote(execDir), JobExecutionUtils.JOB_WRAPPER_SCRIPT);
```

Comment:
Are there ever times where single quoting directories passed into cd will cause unexpected results in bash environments?